packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,149 +8,201 @@ showTitle: true
 
 #### When to use
-Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
+Use an expandable data list when:
+* You have more information than will comfortably fit inside a row.
+* You want to provide a way for advanced users to access information that is not applicable to all users.
+
+## Tables 
+
+### How to use
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
-## Tables
-* **Standard table:** Combines the styling of a list view with behaviors that are commonly associated with a table (sometimes referred to as a hybrid list)
-* **Compact table:** Maximizes the amount of data that can be displayed in a small space
+### Compact and default table
+PatterFly supports two main types of tables:
 
-### Standard table
+1. **[Compact table](#compact-and-default-table)** - when you want to show as much data per page as possible.
+2. **[Default table](#compact-and-default-table)** - when you don’t have to minimize paging.
 
-![standard table](./img/standard-data-table.png)
+See [when to use compact vs. default spacing](#when-to-use-compact-vs-default-spacing) for more information about the styling and usage.
+
+<img src="./img/compact-and-default-table.png"  alt="Compact and default table"  width="857"/> 
+
+### Table elements
+The elements mentioned below are similar for a table with compact or default spacing. The example shows a table with a compact spacing. See [when to use compact vs. default spacing](#when-to-use-compact-vs-default-spacing) for more information about the styling and usage. 
+
+<img src="./img/compact-data-table.png"  alt="Compact table"  width="1187"/> 
 
 1. **[Toolbar](/design-guidelines/usage-and-behavior/toolbar):** Sits above the table and contains controls for manipulating table data. Common actions include filtering, sorting, and pagination.
-2. **Select all:** When present, selects all items in a table. If pagination is being used, this will only select items on the current page.
-3. **Column headers:** Should align with the content they contain. If the user is able to sort on a column, the first click on the header will sort the content of the table on the content in that column. Subsequent clicks will toggle the direction of the sort. Table data can only be sorted on one column at a time. See [Sorting by columns](#sorting-by-columns) for more information on the sort component.
+2. **[Select all](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page.
```

Comment:
In the library we are now calling the "Bulk selection." But I wonder if should even talk about this here as we now have a separate document that describes the bulk selection behavior https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/bulk-selection Maybe reference that? Should the Data list section also reference this, too (and include in mock)?